src/variableProcessor.ts

```diff
@@ -1,14 +1,19 @@
 'use strict';
+import { VariableType } from "./models/variableType";

```

Comment:
Adjust the import order, move them in regions of custom import

In my code structure, the imports are sorted as below:
1. VS Code imports
2. Custom imports (From our extension internal code)
3. 3rd party libraries import

So could you please resort all your files? Thanks in advance.